### DISCUSSION TITLE
ScriptReference/Editor.CreateEditor

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#39] [Notes]
# ScriptReference/Editor.CreateEditor

https://docs.unity3d.com/ScriptReference/Editor.CreateEditor.html

**Comment by vertxxyz** [comment]
Although mentioned, it is not present in all cleanup in the example: it's important to [`DestroyImmediate`](https://docs.unity3d.com/ScriptReference/Object.DestroyImmediate.html) the created Editor.
If you are using this method and getting errors from an associated Editor's OnEnable, it is likely that you have missed doing this in an **OnDisable** method—or if relevant—before recreating the Editor.
